lib/onelogin/ruby-saml/response.rb

```diff
@@ -595,12 +597,14 @@ def validate_in_response_to
       end
 
       # Validates the Audience, (If the Audience match the Service Provider EntityID)
+      # If the response was initialized with the :skip_audience option, this validation is skipped,
       # If fails, the error is added to the errors array
       # @return [Boolean] True if there is an Audience Element that match the Service Provider EntityID, otherwise False if soft=True
       # @raise [ValidationError] if soft == false and validation fails
       #
       def validate_audience
         return true if audiences.empty? || settings.sp_entity_id.nil? || settings.sp_entity_id.empty?
+        return true if options[:skip_audience]
```

Comment:
can you do it before anything else?

Reply:
@pitbulk Sure, I was following the format done in `#validate_destination`, otherwise I would have defaulted to it being first check.

Reply:
@pitbulk done 👍